integrations/Clevertap/browser.js

```diff
@@ -117,7 +117,6 @@ class Clevertap {
     Object.keys(payload).map((key) => {
       if (isObject(payload[key])) {
         logger.debug("cannot process, unsupported traits");
-        return;
```

Comment:
Why return statements are removed?

Reply:
It's an eslint fix. It doesn't make sense to have an empty return statement.